docs/designdoc/clustering_stop.md

```diff
@@ -71,27 +74,70 @@ status:
       lastTransitionTime: 2018-01-01T00:00:00Z
 ```
 
-Also, `STOPPED` will be added to the table displayed when you do `kubectl get mysqlcluster`.
+Also, `CLUSTERING STOPPED` will be added to the table displayed when you do `kubectl get mysqlcluster`.
 
 ```console
 $ kubectl get mysqlcluster
-NAME   AVAILABLE   HEALTHY   STOPPED   PRIMARY   SYNCED REPLICAS   ERRANT REPLICAS   LAST BACKUP
-test   Unknown     Unknown   True      0         2                                   <no value>
+NAME   AVAILABLE   HEALTHY   PRIMARY   SYNCED REPLICAS   ERRANT REPLICAS   CLUSTERING STOPPED   RECONCILE STOPPED   LAST BACKUP
```

Comment:
Is this correct?
The following section uses "CLUSTERING ACTIVE" and "RECONCILE ACTIVE".

Reply:
Thanks 🙏 
I fixed: https://github.com/cybozu-go/moco/pull/576/commits/06e8204b96fddc01398fcca7bd3cabe613cdb902